packages/manager/apps/hycu/src/utils/downloadTextAsFile.ts

```diff
@@ -0,0 +1,13 @@
+/* c8 ignore start */
```

Comment:
Should not be excluded from the coverage report IMO.
By the way, it's better to use c8 configuration file instead